lib/orderbook/OrderBookRepository.ts

```diff
@@ -40,6 +44,24 @@ class OrderbookRepository {
       await this.models.Order.create(order as OrderAttributes);
     }
   }
+
+  /**
+   * Only persist multiple orders if they don't exists.
+   * @param orders array of orders to persist
+   */
+  public addOrdersIfNotExists = async (orders: db.OrderFactory[]) => {
+    const ordersToAdd: Promise<void>[] = [];
```

Comment:
`ordersToAdd` is not  exactly descriptive - we might not be adding anything if the orders already exist for instance. Maybe just call this `orderPromises` or `addOrderPromises`?